### ISSUE TITLE
No way to add accessibility attributes

### ISSUE THREAD

**Issue by marutypes** [nikkomiu/phoenix_inline_svg#12]
Hey, really enjoying using this library in my Phoenix app, but unfortunately I noticed that there's an issue with trying to get my SVGs to play nicely with screen readers.

Though the HTML attribute logic in [`append_html_attribute`](https://github.com/nikkomiu/phoenix_inline_svg/blob/master/lib/phoenix_inline_svg/helpers.ex#L202) looks like it should work fine for any random HTML attribute, the wrapping function `apply_opt` is only setup for `class` and `id` in particular.

This leaves us with no way to setup accessibility attributes such as `aria-labelledby` or `role` without building something hacky on top of the library.

This can also come up as an issue if attempting to use `data` attributes to hook into with javascript.

**TLDR: `svg_image` should allow arbitrary HTML attributes, or at least `role` and `aria-labelledby`**

**Comment by nikkomiu** [comment]
Thank you for the very detailed use case for adding extra attributes. I agree that it would make a lot of sense to get those (as well as allowing any arbitrary attributes) added.

I won't have time to add them into the library for the next few weeks. However, if you want to try to get it added I would take a PR for it!

**Comment by mhanberg** [comment]
@TheMallen @nikkomiu PR #13 should address this

**Comment by nikkomiu** [comment]
PR #13 is merged to master. @TheMallen do you want to test against the master branch of the repo?

**Comment by mhanberg** [comment]
I tested this, and it still needs to be able to convert snake case keyword list keys `aria_labelledby` into kebab-case attributes `aria-labelledby`.

I can write a fix for that then this is good to go.

**Comment by mhanberg** [comment]
I believe this can be closed now.

**Comment by marutypes** [comment]
Ahh, sorry I missed all this activity, I had gone with some hacks for my project. Guess I can remove those :) thanks for the quick action!

**Comment by nikkomiu** [comment]
Released under v1.3.0
